Scripts/UI/GameMenu.gd

```diff
@@ -0,0 +1,48 @@
+extends CanvasLayer
```

Comment:
you will need to set process on this to always~
If you are in build mode, the game menu buttons are not functional